packages/ui/stories/core/tag-field.story.tsx

```diff
@@ -3,6 +3,7 @@ import type { Meta, Story } from '@storybook/react';
 import { TagType } from '../../src/core/tag-field/tag-field.props';
 import _TagField, { TagFieldProps } from '../../src/core/tag-field';
 import { AddTagType, Size } from '../../src/core/tag-field/types';
+import { InfoOutline } from '@scaleflex/icons';
```

Comment:
import this icon before import { AddTagType, Size } from '../../src/core/tag-field/types'; 
because it makes error